Assignment/solution_student/css/style.css

```diff
@@ -1,27 +1,193 @@
 /*
 Project Name: HTML Template
 Client: Your Client
-Author: Your Name
+Author: Carly Brandess
 Developer: @poopsplat in Chicago
 */
 
-/******************************************
-/* BASE STYLES
+/* SETUP
 /*******************************************/
+/* Box Model Hack: http://www.paulirish.com/2012/box-sizing-border-box-ftw/ */
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+html {
+  box-sizing: border-box;
+}
+/* Micro clearfix hack: http://nicolasgallagher.com/micro-clearfix-hack/ */
+.clearfix {
+  *zoom: 1;
+}
+.clearfix:before,
+.clearfix:after {
+    content: " ";
+    display: table;
+}
+.clearfix:after {
+  clear: both;
+}
+
+a:hover {
+    text-decoration: none;
+}
+
+header { 
+ 	background-image: url("../images/header_bg.jpg");
+	width: 100%;
+	height: 500px;
+	background-size: contain;
+	background-repeat: no-repeat;
+	margin: 0;
+  	padding: 0;
+}
+
+h1 {
+	color: white;
+	font-weight: bolder;
+	font-size: 24px;
+	padding-left: 20px;
+	padding-top: 5px;
+}
+
+h2 {
+	color: white;
+	font-weight: semibold;
+	font-size: 14px;
+	border-radius: 5px;
+	text-align: center;
+	padding-top: 100px;
+}
+
+h3 {
+	color: #033048;
+	font-size: 20px;
+	font-weight: semibold;
+	padding: 0;
+	margin: 0;
+}
+
+.container {
+	padding: 0;
+	padding-bottom: 20px;
+	margin: 0;
+	width: 50%;
+	text-indent: 20px;
```

Comment:
I think you're intention here was to space the text out away from the edge of the container, but `text-indent` just works to indent the first line of a paragraph, and should be used only for that, not for global spacing like this, which should be done with padding. So you could have done something like this here:
```
padding: 0 20px 20px; 
```
(replacing the other padding declarations you made here) - which says, 0 padding-top, 20px padding left & right, and 20px bottom.